mods/cnc/maps/twist-of-fate/twist-of-fate.lua

```diff
@@ -0,0 +1,263 @@
+						a.Move(CPos.New(48,51))
+						a.Deploy()
+					end
+				end)
+				Utils.Do(ClearPathCameras, function(c)
+					c.Destroy()
+				end)
+			end)
+		end)
+	end)
+
+	NodAirSupport = Actor.Create("Astk.proxy", true, { Owner = EnemyAi })
+	Trigger.AfterDelay(DateTime.Seconds(AstkDelay), function()
+		SendNodAirstrike(true)
+		Trigger.AfterDelay(DateTime.Seconds(20), function()
+			Media.DisplayMessage(UserInterface.Translate("air-strikes-intel-report"))
+			Trigger.AfterDelay(DateTime.Seconds(5), function()
+				CaptureCommCenter = AddSecondaryObjective(Player1, "capture-nod-communications-center")
+			end)
+		end)
+	end)
+
+	Trigger.OnKilledOrCaptured(NodOutPostCYard, function()
+		SendNodAirstrike(false)
+	end)
+
+	Utils.Do(NodGuns, function(g)
+		Trigger.OnKilled(g, function()
+			SendEngTeam()
+			Utils.Do(NodGuns, function(gun) if not gun.IsDead then Trigger.Clear(gun, "OnKilled") end end)
+		end)
+	end)
+
+	Trigger.OnEnteredFootprint(CenterNodBaseEntrance, function(a, id)
+		if a.Owner == Player1 then
+			local cargo = Reinforcements.ReinforceWithTransport(EnemyAi, "tran", { "rmbo" }, { CPos.New(0,32), waypoint12.Location }, { CPos.New(0,32) })[2]
+			Trigger.OnIdle(cargo[1], function()
+				Trigger.Clear(cargo[1], "OnIdle")
+				Media.PlaySpeechNotification(Player1, "BaseAttack")
+				Trigger.AfterDelay(5, function() GoDemolitionMan(cargo[1]) end)
+			end)
+			Trigger.RemoveFootprintTrigger(id)
+		end
+	end)
+
+	Trigger.OnDamaged(NodCYard, function(_, atk, _)
+		if atk.Owner == Player1 and not NukeLaunched then
+			NukeLaunched = true
+			LaunchNuke(false)
+			Trigger.Clear(NodCYard, "OnDamaged")
+			Trigger.AfterDelay(1, function()
+				RepairBuilding(EnemyAi, NodCYard, 0.75)
+				NodCYard.StartBuildingRepairs()
+			end)
+		end
+	end)
+
+	Trigger.OnKilledOrCaptured(NodTmpl, function()
+		NukeDelay = 0
+	end)
+
+	Trigger.OnCapture(NodAstkHq, function()
+		Player1.MarkCompletedObjective(CaptureCommCenter)
+		Trigger.ClearAll(NodAstkHq)
+		AstkDelay = 0
+		Trigger.Clear(NodCYard, "OnDamaged")
+		Trigger.AfterDelay(1, function()
+			RepairBuilding(EnemyAi, NodCYard, 0.75)
+			NodCYard.StartBuildingRepairs()
+		end)
+		Media.DisplayMessage(UserInterface.Translate("communications-center-captured-sams-located"))
+		local activeSams = EnemyAi.GetActorsByType("sam")
+		local miniCams = { }
+		Utils.Do(activeSams, function(s)
+			table.insert(miniCams, Actor.Create("camera.mini", true, { Owner = Player1, Location = s.Location }))
+			table.insert(miniCams, Actor.Create("camera.mini", true, { Owner = Player1, Location = CPos.New(s.Location.X + 1, s.Location.Y) }))
+		end)
+		Trigger.AfterDelay(1, function()
+				Utils.Do(miniCams, function(c)
+					c.Destroy()
+				end)
+			end)
+		Trigger.AfterDelay(200, function()
+			DestroySams = AddSecondaryObjective(Player1, "destroy-sams")
+		end)
+		Trigger.OnAllKilled(NodSams, function()
+			AirSupport = Actor.Create("Astk.proxy", true, { Owner = Player1 })
+			Player1.MarkCompletedObjective(DestroySams)
```

Comment:
I can sympathize with the git goof. Not sure if you made other changes but it looks like the last crash fix was reverted.
```suggestion
		if DestroySams then
			Player1.MarkCompletedObjective(DestroySams)
		end
```